processor/src/test/kotlin/com/tobrun/datacompat/DataCompatProcessorTest.kt

```diff
@@ -15,14 +15,16 @@ private val simpleTest = SourceFile.kotlin(
 """
 import com.tobrun.datacompat.annotation.DataCompat
 
+interface EmptyInterface
 /**
  * Represents a person.
  * @property name The full name.
  * @property nickname The nickname.
  * @property age The age.
  */
+@Deprecated
```

Comment:
Could you add values to the annotation to make sure those are also retained
```suggestion
@Deprecated("Use the new method", level = DeprecationLevel.ERROR)
```

Reply:
for now they are not retained, will check how hard it is to retain. In any case `@Parcelize` does not have values 😄

Reply:
not an easy task assuming there could be N parameters while user has populated only M < N where some could have named parameter etc; guess it's achievable but we don't need it now so postponing

Reply:
ok, thanks for checking.